From f3360e77850f7bf3d2fa1ea9367934663fe564d1 Mon Sep 17 00:00:00 2001
From: Liu Ying <b17645@freescale.com>
Date: Wed, 17 Nov 2010 13:48:12 +0800
Subject: [PATCH 460/670] ENGR00133795-1 FSL Dev:Add CVBS support for TVin
 platform data

This patch adds CVBS support for TVin platform data.

Signed-off-by: Liu Ying <b17645@freescale.com>
---
 include/linux/fsl_devices.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index d1faf3a..5d78280 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -311,6 +311,7 @@ struct mxc_tvin_platform_data {
 	char *pvdd_reg;
 	void (*pwdn) (int pwdn);
 	void (*reset) (void);
+	bool cvbs;
 };
 
 struct mxc_epdc_fb_mode {
-- 
1.8.0

